### PR TITLE
changes to add health disclaimer

### PR DESCRIPTION
**Pull request by teryhill**
changes to add health disclaimer in conjunction with #79

### CODE REVIEW

File: acknowledge_license_cert.html [hunk 1]
```diff
@@ -12,7 +12,9 @@
 <h3>License information</h3>  
 <div class="paragraph">   
 Unless otherwise stated, this code is subject to the terms of the <a href="license.txt">
-GNU General Public License version 3 or higher (GPL)</a>, a copy of which is included here for your convenience.
+GNU General Public License version 3 or higher (GPL)</a>, a copy of which is included here for your convenience.<br>
```
Comment (tmccormi) [inline acknowledge_license_cert.html]: Need to change this to MPL

Comment (teryhill) [inline acknowledge_license_cert.html]: see if that works. I am assuming from your post it is called mpl_license.txt instead of license.txt

**Comment by tmccormi** [comment]
looks good.

**Comment by teryhill** [comment]
Keep this open until I hear back from Larry. I will have some changes to the license statements

**Pull request merged**